smtp/client.js

```diff
@@ -85,7 +85,7 @@ Client.prototype =
       if(!(msg instanceof message.Message) 
           && msg.from 
           && (msg.to || msg.cc || msg.bcc)
-          && (msg.text || this._containsInlinedHtml(msg.attachment)))
+          && ((msg.text || '' != null) || this._containsInlinedHtml(msg.attachment)))
```

Comment:
ah. `msg.text !== undefined` is what i meant. double equals is needed so implied equality doesn't take over. i'll make a new branch with these changes. thanks again.